packages/gatsby-remark-autolink-headers/src/index.js

```diff
@@ -21,6 +21,7 @@ module.exports = (
     maintainCase = false,
     removeAccents = false,
     enableCustomId = false,
+    after = true,
```

Comment:
this changes default behaviour (we should not do that unless we do major version bump for the plugin)

Reply:
+1 for not introducing a breaking change